pkg/testing/runner/debian.go

```diff
@@ -106,6 +106,7 @@ func (DebianRunner) Copy(ctx context.Context, sshClient *ssh.Client, logger Logg
 			"failed to remove agent directory before unziping new one: %w. stdout: %q, stderr: %q",
 			err, stdout, stderr)
 	}
+	_, _, _ = sshRunCommand(ctx, sshClient, "sudo", []string{"rm", "-rf", "beats"}, nil)
```

Comment:
Why is this removing a random beats directory? We set the remote path to `$HOME/agent` I don't see why the same cannot be used for beats?

Reply:
So, the working directory is the same here, since tests are still being run out of the agent repo, the only difference is that now we have additional beat artifacts that we upload in addition to the agent working directory. 

We could refactor this so we don't have `/agent` hard-coded, but we'd still need to find a way to delete the working directory and any non-working-directory artifacts. I wasn't too sure what the runner was doing, do I didn't want to blow away the entire home directory or something.

Reply:
I feel like the best solution here is to probably just to make some of the copy code more generic, and just use some kind of heuristic to delete anything that looks like a working directory or artifact.

Reply:
Alright, moved things around to make the code a little more generic, at least.